crates/astria-composer/src/collectors/mod.rs

```diff
@@ -1,9 +1,30 @@
 pub(crate) mod geth;
 pub(crate) mod grpc;
 
+use std::{
+    fmt::{
+        Display,
+        Formatter,
+    },
+    time::Duration,
+};
+
 const EXECUTOR_SEND_TIMEOUT: Duration = Duration::from_millis(500);
 
-use std::time::Duration;
+#[derive(Debug)]
+pub(crate) enum CollectorType {
```

Comment:
This has no advantage over using ` const COLLECTOR_TYPE: &str` in `geth` and `grpc`, respectively. Just do the const, it's simpler.